lib/codelog/command/release.rb

```diff
@@ -1,12 +1,19 @@
 module Codelog
   module Command
     class Release
+      include FileUtils
```

Comment:
@Guilherme2112 Why do you need `FileUtils` here?

Reply:
@emilianoLeite Not really, removed on  4e751d15f61fdc59c935c442046d0740a4779d69